### PR TITLE
Grid

### PR DESCRIPTION
**Pull request by nikhilaravi**
- Adds react, redux set up
- Adds Masonry grid set up

https://tc-jira.atlassian.net/browse/ISEARCH-191

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,49 +1,63 @@
...
+    "imports-loader": "^0.6.5"
   },
   "dependencies": {
     "react": "^0.14.7",
-    "react-dom": "^0.14.7"
+    "react-dom": "^0.14.7",
+    "react-masonry-component": "^4.0.0",
+    "react-redux": "^4.4.0",
+    "redux": "^3.3.1",
+    "material-ui": "^0.14.4",
```
Comment (nelsonic) [inline package.json]: Trailing comma on line 60 means JSON not valid so `npm install` fails.

**Comment by nelsonic** [comment]
Most of this PR is *super*!
Sadly, the `lint` command fails ...

**Comment by nikhilaravi** [comment]
@nelsonic okay I've fixed the lint errors now!

**Comment by nelsonic** [comment]
@nikhilaravi thanks for fixing. waiting for travis. :+1:

**Comment by nelsonic** [comment]
@nikhilaravi, sadly Travis is not happy: https://travis-ci.org/numo-labs/isearch-ui/jobs/114193159#L462
do you have time to investigate/address it?

**Comment by nikhilaravi** [comment]
@nelsonic yes sure I'll fix it now!

**Comment by nikhilaravi** [comment]
@nelsonic this should now be fixed! I had to add a null css mocha compiler.

**Comment by nelsonic** [comment]
Nice one. Thanks! :+1:

**Pull request merged**